### DISCUSSION TITLE
SendTransaction not working

### DISCUSSION THREAD

**Discussion by KetulCodiste** [Web3Auth/Web3Auth#461] [Help]
I have implemented the React + web3Auth demo from the web3Auth documentation. All the functions are working fine except the sendTransaction function. It gives me the following error: 

Error: Internal JSON-RPC error.
    at getEthJsonRpcError (errors.ts:188:1)
    at Object.internal (errors.ts:52:1)
    at provider.sendAsync (INetworkController.ts:125:1)
    at async provider.request (INetworkController.ts:149:1)

I also tried to implement one more function which uses .send() method. So, .call() method is working correctly but .send() method is giving this error. 
I have not made any changes in the demo, just copy/pasted the code in my local machine.

**Comment by himanshuchawla009** [comment]
can you paste the code of your send function here, also plz check if rpc network call is failing then plz check the response of failed network call.

**Reply by KetulCodiste** [reply]
I tried a different demo now, it is working. Thank you

**Reply by gaiverrr** [reply]
@KetulCodiste can you pls provide link to working demo?

**Reply by KetulCodiste** [reply]
@gaiverrr https://github.com/Web3Auth/Web3Auth/tree/master/demo/react-app
